dev/com.ibm.websphere.appserver.features/visibility/public/sessionDatabase-1.0/com.ibm.websphere.appserver.sessionDatabase-1.0.feature

```diff
@@ -7,13 +7,10 @@ Manifest-Version: 1.0
 Subsystem-Name: Database Session Persistence 1.0
 -features=com.ibm.websphere.appserver.sessionStore-1.0.0.Database, \
  com.ibm.websphere.appserver.jndi-1.0, \
- io.openliberty.servlet.api-3.0; ibm.tolerates:="3.1,4.0"; apiJar=false, \
- com.ibm.websphere.appserver.transaction-1.1; ibm.tolerates:=1.2, \
+ io.openliberty.servlet.api-3.0; ibm.tolerates:="3.1,4.0,5.0"; apiJar=false, \
+ com.ibm.websphere.appserver.transaction-1.1; ibm.tolerates:="1.2,2.0", \
  com.ibm.websphere.appserver.jdbc-4.0; ibm.tolerates:="4.1, 4.2, 4.3"
 -bundles=com.ibm.websphere.security, \
- com.ibm.ws.session.store, \
- com.ibm.ws.session.db, \
- com.ibm.ws.serialization, \
- com.ibm.ws.session
```

Comment:
There was no `com.ibm.websphere.javaee.jcache.1.1` in the original `sessionDatabase-1.0` feature.  It looks like the jakarta version of it is not needed in the auto feature `io.openliberty.sessionDatabase1.0.jakarta`